ncj/vray/render-linux-with-blobfuse-mount/job.template.json

```diff
@@ -0,0 +1,158 @@
+{
+    "templateMetadata": {
+        "description": "Application template for working with V-Ray standalone renderer on CentOS."
+    },
+    "parameters": {
+        "poolId": {
+            "type": "string",
+            "defaultValue": "vray-pool-centos",
+            "metadata": {
+                "description": "The ID of the pool on which to run the job."
+            }
+        },
+        "jobName": {
+            "type": "string",
+            "defaultValue": "render-vray",
+            "metadata": {
+                "description": "The unique name of the job."
+            }
+        },
+        "inputFilegroupName": {
+            "type": "string",
+            "metadata": {
+                "description": "The file group where the data is stored",
+                "advancedType": "file-group"
+            }
+        },
+        "sceneFile": {
+            "type": "string",
+            "metadata": {
+                "description": "The V-Ray scene file to be rendered (.vrscene)",
+                "advancedType": "file-in-file-group",
+                "dependsOn": "inputFilegroupName"
+            }
+        },
+		"inputFilegroupSas": {

```

Comment:
Whitespace...

Reply:
done and sorted the `commandline` as well. :)